### PR TITLE
MAT-3190: Adding menu to layout for public site

### PR DESCRIPTION
**Pull request by gregory-akins**

### CODE REVIEW

File: src/styles/styles.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
+/** @format */
```
Comment (adongare) [inline src/styles/styles.tsx:1]: does it make sense to ignore styles from test cases(jest config)?

**Review by adongare — DISMISSED** [review]

File: package.json [hunk 2]
```diff
@@ -16,7 +16,8 @@
     "watch-tests": "cross-env BABEL_ENV=test jest --watch",
     "prepare": "husky install",
     "coverage": "cross-env BABEL_ENV=test jest --coverage",
-    "build:types": "tsc"
+    "build:types": "tsc",
+    "tailwind:css": "tailwind build src/tailwind.css -c tailwind.config.js -o src/index.css"
```
Comment (AndrewBird81) [inline package.json:20]: We shouldn't need this (or the tailwind.css file, or the generated index.css)

File: src/root.component.tsx [hunk 3]
```diff
@@ -1,9 +1,29 @@
+/** @format */
+
 import React, { useState } from "react";
-import tw from "twin.macro";
 import GlobalStyles from "./styles/GlobalStyles";
 import { MadieEditor } from "@madie/madie-editor";
+import { tsVoidKeyword } from "@babel/types";
```
Comment (AndrewBird81) [inline src/root.component.tsx:6]: We don't need this unused import

File: src/root.component.tsx [hunk 4]
```diff
@@ -1,9 +1,29 @@
...
+import tw, { css, styled, theme } from "twin.macro";
```
Comment (AndrewBird81) [inline src/root.component.tsx:7]: tw, css, and theme aren't used

File: src/root.component.tsx [hunk 5]
```diff
@@ -19,9 +39,97 @@ export default function Root(props) {
...
     <>
       <GlobalStyles />
-      <Notice>{props.name} is mounted!</Notice>
       <MadieEditor {...outputProps} />
       <div data-testid="madie-editor-value">{editorVal}</div>
+      <Nav>
+        <InnerNav>
+          <InnerMost>
+            <Logo href="/">
+              <Image>
```
Comment (AndrewBird81) [inline src/root.component.tsx:48]: I think ideally the SVG would be a separate file, imported via webpack

**Review by AndrewBird81 — CHANGES_REQUESTED** [review]

File: src/styles/styles.tsx (hunk 1 above)
Comment (gregory-akins) [reply src/styles/styles.tsx:1]: These styles are used by components in the page, so while the file was a way to seperate a list of styles, they're still part of the objects required to render root.component.tsx

File: src/root.component.tsx (hunk 3 above)
Comment (gregory-akins) [reply src/root.component.tsx:6]: Removed it.

File: src/root.component.tsx (hunk 5 above)
Comment (gregory-akins) [reply src/root.component.tsx]: Done

File: src/root.component.tsx (hunk 4 above)
Comment (gregory-akins) [reply src/root.component.tsx:7]: removed

File: package.json (hunk 2 above)
Comment (gregory-akins) [reply package.json]: removed

**Review by AndrewBird81 — DISMISSED** [review]

**Review by AndrewBird81 — APPROVED** [review]

**Pull request merged**